src/EventId.php

```diff
@@ -9,12 +9,12 @@
  *
  * @author Stefano Arlandini <sarlandini@alice.it>
  */
-final class EventId
+class EventId
```

Comment:
I would prefer to not do this. This would mean being able to use a `SpanId` in place of an `EventId`, which should not be possible.